### DISCUSSION TITLE
Question about functionality

### DISCUSSION THREAD

**Discussion by Tiwas** [Taxel/PlexTraktSync#1430] [General]
I didn't find anything specific about this, so I'm daring to ask about one functionality I'm missing.

So...I got PlexTraktSync up and running, and it's syncing. So far, so good. After the sync, I see all my collections (aka "things I've watched in the past") are synced, but - my watch history isn't updated. Is that outside the scope of this project? That's really the most interesting functionality for me :)

Thanks.

**Comment by simonc56** [comment]
Be more specific in title and in first post.
"_my collections_" and "_my watch history_" is too vague.
Also show config file and output of last sync.

**Reply by Tiwas** [reply]
Sorry about that. I can get the sync output when I'm home, but where's the config?

My watch history is the one located at https://trakt.tv/users/username/history . When I've used scrobbling before, a show or movie would be listed here on the date it was actually viewed.
